mlflow/server/js/src/experiment-tracking/components/ExperimentListView.js

```diff
@@ -204,8 +212,11 @@ export class ExperimentListView extends Component {
               treeData={treeData}
               dangerouslySetAntdProps={{
                 selectable: true,
+                checkable: true,
                 multiple: true,
-                selectedKeys: [activeExperimentId],
+                selectedKeys: activeExperimentIds,
+                checkedKeys: activeExperimentIds,
+                onCheck: this.handleCheck(),
```

Comment:
```suggestion
                onCheck: this.handleCheck,
```

Reply:
will change. I think I'll have to do that through my local repo copy instead of just clicking on "Commit suggestion" button, otherwise the PR fails saying the commit is not signed (unless there is a way to also sign the commit done through "Commit suggestion" button